src/reducers/contacts/contactReducer.js

```diff
@@ -4,50 +4,48 @@ const INITIAL_STATE = {
   contacts: [],
   currentContact: {},
   addingContact: true,
-  contactName: "",
-  contactAddress: {}
-}
+  contactName: '',
+  contactAddress: {},
+};
 
 /**
- * Reducer used to handle all actions occurring during the process of 
+ * Reducer used to handle all actions occurring during the process of
  * adding a new contact. Each case represents an Action
  */
 export default (state = INITIAL_STATE, action) => {
   switch (action.type) {
     case actions.ADDING_CONTACT:
       var contact = state.currentContact;
 
-      //Object.keys(contact)[0]
+      // Object.keys(contact)[0]
```

Comment:
@CFDevelopment r these unused code?

Reply:
comments mean unused yes, I will verify and remove them

Reply:
fixed